cmd/trace-agent/main_windows.go

```diff
@@ -66,87 +76,32 @@ func (m *myservice) Execute(args []string, r <-chan svc.ChangeRequest, changes c
 	return
 }
 
-func runService(isDebug bool) {
+func runService() {
 	var err error
-	if isDebug {
-		elog = debug.New(ServiceName)
-	} else {
-		elog, err = eventlog.Open(ServiceName)
-		if err != nil {
-			return
-		}
+	elog, err = eventlog.Open(ServiceName)
+	if err != nil {
+		return
 	}
 	defer elog.Close()
 
-	run := svc.Run
-	if isDebug {
-		run = debug.Run
-	}
 	elog.Info(0x40000007, ServiceName)
-	err = run(ServiceName, &myservice{})
+	err = svc.Run(ServiceName, &myservice{})
 	if err != nil {
 		elog.Error(0xc0000008, err.Error())
 		return
 	}
 	elog.Info(0x40000004, ServiceName)
 }
 
-// main is the main application entry point
-func main() {
+func runAgent() {
 	isIntSess, err := svc.IsAnInteractiveSession()
 	if err != nil {
 		fmt.Printf("failed to determine if we are running in an interactive session: %v", err)
 	}
 	if !isIntSess {
-		runService(false)
-		return
-	}
-	// sigh.  Go doesn't have boolean xor operator.  The options are mutually exclusive,
-	// make sure more than one wasn't specified
-	optcount := 0
-	if flags.Win.InstallService {
```

Comment:
Are we breaking anything on Windows, or all of this has been addressed differently?

Reply:
They are different commands now. I'm hoping @derekwbrown will take a look to ensure all is well. I'll also be trying to install a local `.msi` on my VM.